### ISSUE TITLE
specifying only an upper bound in GROUP BY time() with too many point yields no error

### ISSUE THREAD

**Issue by rkuchan** [influxdata/influxdb#4004]
When I enter a `GROUP BY time()` query with too many points and I only specify a lower bound in the `WHERE` clause I get the proper error:
```
> select mean(bikes_available) from bikes where time > 0 group by time(1s)
ERR: too many points in the group by interval. maybe you forgot to specify a where time clause?
```

When I enter a `GROUP BY time()` query with too many points and I specify a lower bound and an upper bound in the `WHERE` clause I get the proper error:
```
> select mean(bikes_available) from bikes where time > 0 and time < now() group by time(1s)
ERR: too many points in the group by interval. maybe you forgot to specify a where time clause?
```

When I enter a `GROUP BY time()` query with too many points and I only specify an upper bound in the `WHERE` clause I do not get the proper error:
```
> select mean(bikes_available) from bikes where time < now() group by time(1s)
name: bikes
-----------
time			mean
1970-01-01T00:00:00Z
```

I see that Michael experienced the same issue in: https://github.com/influxdb/influxdb/issues/2702#issuecomment-117333972

**Comment by zstyblik** [comment]
This is still happening with 0.9.6.1.

**Comment by BrannonKing** [comment]
+1. This is bad. New users like myself have no way to know what is going on.

**Comment by jsternberg** [comment]
This is fixed in 0.12 since the old bucket limit was removed and a new (optional) one was added that works differently. Thanks!
